charts/gadget/templates/ocistorepv.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: oci-store-pv
+spec:
+  capacity:
+    storage: {{- .Values.persistentVolume.storageCapacity }}  
+  accessModes:
+    - ReadWriteOnce  
+  local:
+    path: /var/lib/ig/oci-store  
```

Comment:
😅 I finally understood what you meant by using local directly. Apologies! Yes I think we can use it directly instead of hostPath. I'll push the changes.